_episodes/51-managing-software.md

```diff
@@ -1,4 +1,4 @@
----
+x---
```

Comment:
```suggestion
---
```

Reply:
Do you need to commit this suggested change?

Reply:
hmmm a buggy UI? I re-did it now